library/network/src/modules/SuSEFirewall.rb

```diff
@@ -1320,6 +1320,54 @@ def SetServicesForZones(services_ids, firewall_zones, new_status)
       nil
     end
 
+    # Function returns actual state of Masquerading support.
+    # In FirewallD, masquerade is enabled per-zone so this
+    # function has to iterate through all the zones and return
+    # a collective boolean value. This API is probably not very
+    # useful for FirewallD
+    #
+    # @param    zone to get masqurade status from
+    # @return	[Boolean] if supported
+    def GetMasquerade(zone = nil)
+      if zone.is_a?(::String)
```

Comment:
In cases where we have a nullable type, for example "String or nil", it is common to check for `nil`, not for the concrete class. So `if !zone.nil?`